bucket/arduino.json

```diff
@@ -34,7 +34,7 @@
         "regex": "Arduino IDE ([\\d.]+)<"
     },
     "autoupdate": {
-        "url": "https://downloads.arduino.cc/arduino-$version-windows.zip",
+        "url": "https://downloads.arduino.cc/arduino-ide/arduino-ide_$version_Windows_64bit.zip",
         "hash": {
             "url": "$baseurl/arduino-$version.sha512sum.txt"
         },
```

Comment:
```suggestion
```
Remove hash from autoupdate as hashes are no longer provided on the website. (Hashes on GitHub releases are only for the .exe download, sadly)